keyboards/crkbd/rev2/info.json

```diff
@@ -0,0 +1,233 @@
+            "breathing": true,
+            "band_sat": true,
+            "band_val": true,
+            "band_pinwheel_sat": true,
+            "band_pinwheel_val": true,
+            "band_spiral_sat": true,
+            "band_spiral_val": true,
+            "cycle_all": true,
+            "cycle_left_right": true,
+            "cycle_up_down": true,
+            "rainbow_moving_chevron": true,
+            "cycle_out_in": true,
+            "cycle_out_in_dual": true,
+            "cycle_pinwheel": true,
+            "cycle_spiral": true,
+            "dual_beacon": true,
+            "rainbow_beacon": true,
+            "rainbow_pinwheels": true,
+            "raindrops": true,
+            "jellybean_raindrops": true,
+            "hue_breathing": true,
+            "hue_pendulum": true,
+            "hue_wave": true,
+            "pixel_fractal": true,
+            "pixel_flow": true,
+            "pixel_rain": true,
+            "typing_heatmap": true,
+            "digital_rain": true,
+            "solid_reactive_simple": true,
+            "solid_reactive": true,
+            "solid_reactive_wide": true,
+            "solid_reactive_multiwide": true,
+            "solid_reactive_cross": true,
+            "solid_reactive_multicross": true,
+            "solid_reactive_nexus": true,
+            "solid_reactive_multinexus": true,
+            "splash": true,
+            "multisplash": true,
+            "solid_splash": true,
+            "solid_multisplash": true
+        },
+        "layout": [
+            {"matrix": [3, 5], "x": 95, "y": 63, "flags": 4},
+            {"matrix": [2, 5], "x": 85, "y": 39, "flags": 4},
+            {"matrix": [1, 5], "x": 85, "y": 21, "flags": 4},
+            {"matrix": [0, 5], "x": 85, "y": 4, "flags": 4},
+            {"matrix": [0, 4], "x": 68, "y": 2, "flags": 4},
+            {"matrix": [1, 4], "x": 68, "y": 19, "flags": 4},
+            {"matrix": [2, 4], "x": 68, "y": 37, "flags": 4},
+            {"matrix": [3, 4], "x": 80, "y": 58, "flags": 4},
+            {"matrix": [3, 3], "x": 60, "y": 55, "flags": 4},
+            {"matrix": [2, 3], "x": 50, "y": 35, "flags": 4},
+            {"matrix": [1, 3], "x": 50, "y": 13, "flags": 4},
+            {"matrix": [0, 3], "x": 50, "y": 0, "flags": 4},
+            {"matrix": [0, 2], "x": 33, "y": 3, "flags": 4},
+            {"matrix": [1, 2], "x": 33, "y": 20, "flags": 4},
+            {"matrix": [2, 2], "x": 33, "y": 37, "flags": 4},
+            {"matrix": [2, 1], "x": 16, "y": 42, "flags": 4},
+            {"matrix": [1, 1], "x": 16, "y": 24, "flags": 4},
+            {"matrix": [0, 1], "x": 16, "y": 7, "flags": 4},
+            {"matrix": [0, 0], "x": 0, "y": 7, "flags": 4},
+            {"matrix": [1, 0], "x": 0, "y": 24, "flags": 4},
+            {"matrix": [2, 0], "x": 0, "y": 41, "flags": 4},
+            {"matrix": [0, 6], "x": 103, "y": 17, "flags": 4},
+            {"matrix": [1, 6], "x": 103, "y": 24, "flags": 4},
```

Comment:
This looks like an unfortunate PCB misdesign — break-off keys (`[0, 0]`, `[1, 0]`, `[2, 0]`) should have been last in the chain.  As implemented now, breaking off the outer column would also disable the LEDs under the extra inner keys (and you would also be relying on the behavior of LEDs with floating DIN inputs).

Reply:
@sigprof this aspect actually seems to be accounted in the PCB design by including s solder bridge that allows the redirection of the DIN/DOUT signals for the LED chain.

Reply:
The solder bridge solves the hardware part of the problem; however, after closing that bridge the LEDs for the extra inner keys would have wrong indices (not matching the RGB matrix configuration), so the per-key lighting functionality for those keys would be broken unless you patch `g_led_config` appropriately.  Any indicator code would also need changes to take the changed indices into account.

Currently QMK does not offer any good solution in that area apart from defining separate sub-revisions of the PCB with separate `rgb_matrix` configs in `info.json`.

Reply:
Indeed, I guess that the indices issue wasn't considered or simply not regarded as an important aspect. Ofc effects would look weird and all over the place.

Reply:
hmm, you are correct. I thought I could just provide a separately modified and built file for 3x5, but it would be better if it could be expressed in info.json, which it is.

Reply:
How about separating the revision for `rev2_mini` since the number of keys and LEDs are different?

Reply:
I added `rev4_mini`. How about it?